viewer/packages/rendering/src/sector/CadNode.ts

```diff
@@ -177,6 +184,21 @@ export class CadNode extends THREE.Object3D {
     this._instancedMeshManager.removeSectorInstancedMeshes(sectorId);
   }
 
+  public batchGeometry(
+    geometryBatchingQueue: [
+      type: RevealGeometryCollectionType,
+      geometryBuffer: THREE.BufferGeometry,
+      instanceId: string | undefined
+    ][],
+    sectorId: number
+  ) {
```

Comment:
Always add return type of public functions (added as a lint step that soon will hit this)